app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_instances = [Person(p["name"], p["age"]) for p in people]
+
+    for person in people:
+        instance = Person.people[person["name"]]
+
+        if check_marital_status(person) == "married":
+            if check_gender(person) == "man":
+                instance.wife = Person.people[person["wife"]]
+            else:
+                instance.husband = Person.people[person["husband"]]
+
+    return list_of_instances
+
+
+def check_gender(person: dict) -> str:
+    return "man" if "wife" in person else "woman"
+
+
+def check_marital_status(person: dict) -> str:
+    return "married" if person.get("husband") is not None \
```

Comment:
We are not allowed to move the string like this. With using " \ ". Must be like this (
code
code
)
or like this ( code
code)